packages/dart_frog_cli/lib/src/commands/uninstall/uninstall.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:args/command_runner.dart';
+import 'package:dart_frog_cli/src/command_runner.dart';
+import 'package:mason/mason.dart' hide packageVersion;
+
+/// {@template uninstall_command}
+/// `dart_frog uninstall` command which explains how to uninstall
+/// the dart_frog_cli.
+/// {@endtemplate}
+class UninstallCommand extends Command<int> {
+  /// {@macro uninstall_command}
+  UninstallCommand({
+    required Logger logger,
+  }) : _logger = logger;
+
+  final Logger _logger;
+
+  @override
+  String get description => 'Explains how to uninstall the Dart Frog CLI.';
+
+  @override
+  String get name => 'uninstall';
+
+  @override
+  Future<int> run() async {
+    _logger.info(
+      'For instructions on how to uninstall $packageName completely, check out:'
+      '\nhttps://dartfrog.vgv.dev/docs/overview#uninstalling-',
```

Comment:
We should use a link instead, see [this](https://github.com/VeryGoodOpenSource/dart_frog/blob/060c593baa50d9b6489e1cee5300a7277f80160c/packages/dart_frog_cli/lib/src/dev_server_runner/dev_server_runner.dart#L313) as an example.